packages/flutter/test/widgets/focus_manager_test.dart

```diff
@@ -500,13 +486,7 @@ void main() {
       expect(lastMode, isNull);
       focusManager.highlightStrategy = FocusHighlightStrategy.automatic;
       expect(focusManager.highlightMode, equals(FocusHighlightMode.touch));
-      sendFakeKeyEvent(<String, dynamic>{
-        'type': 'keydown',
-        'keymap': 'fuchsia',
-        'hidUsage': 0x04,
-        'codePoint': 0x64,
-        'modifiers': RawKeyEventDataFuchsia.modifierLeftMeta,
-      });
+      await tester.sendKeyEvent(LogicalKeyboardKey.metaLeft, platform: 'fuchsia');
```

Comment:
It probably doesn't matter if the test passes, but the original test only send down event.

Reply:
Yes, the test passes.  I thought about it when I converted this, and I think that the old way was not really testing in a "real world" way: in the real world, it's unlikely to only have down events.